docs/pt/docs/tutorial/metadata.md

```diff
@@ -0,0 +1,132 @@
+| `license_info` | `dict` | As informações de licença para a API exposta. Ela pode conter vários campos. <details><summary>Campos de <code>license_info</code></summary><table><thead><tr><th>Parâmetro</th><th>Tipo</th><th>Descrição</th></tr></thead><tbody><tr><td><code>name</code></td><td><code>str</code></td><td><strong>OBRIGATÓRIO</strong> (se um <code>license_info</code> for definido). O nome da licença usada para a API.</td></tr><tr><td><code>identifier</code></td><td><code>str</code></td><td>Uma expressão de licença <a href="https://spdx.org/licenses/" class="external-link" target="_blank">SPDX</a> para a API. O campo <code>identifier</code> é mutuamente exclusivo do campo <code>url</code>. <small>Disponível desde OpenAPI 3.1.0, FastAPI 0.99.0.</small></td></tr><tr><td><code>url</code></td><td><code>str</code></td><td>Uma URL para a licença usada para a API. DEVE estar no formato de uma URL.</td></tr></tbody></table></details> |
+
+Você pode defini-los da seguinte maneira:
+
+```Python hl_lines="3-16  19-32"
+{!../../docs_src/metadata/tutorial001.py!}
+```
+
+/// tip
+
+Você pode escrever Markdown no campo `description` e ele será renderizado na saída.
+
+///
+
+Com essa configuração, a documentação automática da API se pareceria com:
+
+<img src="/img/tutorial/metadata/image01.png">
+
+## License identifier
+
+Desde o OpenAPI 3.1.0 e FastAPI 0.99.0, você também pode definir o license_info com um identifier em vez de uma url.
+
+Por exemplo:
+
+```Python hl_lines="31"
+{!../../docs_src/metadata/tutorial001_1.py!}
+```
+
+## Metadados para tags
+
+Você também pode adicionar metadados adicionais para as diferentes tags usadas para agrupar suas operações de caminho com o parâmetro `openapi_tags`.
+
+Ele recebe uma lista contendo um dicionário para cada tag.
+
+Cada dicionário pode conter:
+
+* `name` (**obrigatório**): uma `str` com o mesmo nome da tag que você usa no parâmetro `tags` nas suas *operações de caminho* e `APIRouter`s.
+* `description`: uma `str` com uma breve descrição da tag. Pode conter Markdown e será exibido na interface de documentação.
+* `externalDocs`: um `dict` descrevendo a documentação externa com:
+    * `description`: uma `str` com uma breve descrição da documentação externa.
+    * `url` (**obrigatório**): uma `str` com a URL da documentação externa.
+
+### Criar Metadados para tags
+
+Vamos tentar isso em um exemplo com tags para `users` e `items`.
+
+Crie metadados para suas tags e passe-os para o parâmetro `openapi_tags`:
+
+```Python hl_lines="3-16  18"
+{!../../docs_src/metadata/tutorial004.py!}
+```
+
+Observe que você pode usar Markdown dentro das descrições. Por exemplo, "login" será exibido em negrito (**login**) e "fancy" será exibido em itálico (_fancy_).
+
+/// tip
+
+Você não precisa adicionar metadados para todas as tags que você usa.
+
+///
+
+### Use suas tags
+
+Use o parâmetro `tags` com suas *operações de caminho* (e `APIRouter`s) para atribuí-los a diferentes tags:
```

Comment:
Most of the translations use operações de rota instead of operações de caminho. I think it would be nice the keep the pattern 😄